tag_hierarchy/commands/nodes.cpp

```diff
@@ -0,0 +1,138 @@
+//
+// Created by Petter Moe Kvalvaag on 2019-10-03.
+//
+
+#include <tag_hierarchy/filteredhierarchyvisitor.h>
+#include "tag_hierarchy/tag_hierarchy.h"
+#include "tag_hierarchy/commands/nodes.h"
+
+Nodes nodes(std::string("nodes"));
+
+Nodes::Nodes(std::string name) : Command(name) {}
+
+std::vector<NodeType>
+Nodes::ProcessRequest(std::vector<NodeType> &nodes)
```

Comment:
Please consider breaking up

Reply:
This is actually being refactored in another branch, so that the filter settings get reusable. I will not address it for this review to avoid merge conflicts.